lib/diagnostic.js

```diff
@@ -9,14 +9,26 @@
 // Write a callback for reduce that selects
 // the minimum value in the array
 const min = function min() {
+  let answer = arguments[0];
+  for (let i = 1; i < arguments.length; i++) {
+    if (answer > arguments[i]) {
+      answer = arguments[i];
+    }
+  }
 
+  return answer;
```

Comment:
We were actually looking for you to write a callback function that you could pass into reduce as an argument; in other words,
```
[1,2,3].reduce(min)
// => 1
```